src/main/java/net/fabricmc/loader/metadata/CustomValueImpl.java

```diff
@@ -41,7 +41,7 @@
 	public static CustomValue fromJsonElement(JsonElement e) {
 		if (e instanceof JsonObject) {
 			JsonObject o = (JsonObject) e;
-			Map<String, CustomValue> entries = new LinkedHashMap<>(o.size());
+			Map<String, CustomValue> entries = new LinkedHashMap<>(o.entrySet().size());
```

Comment:
Why is this needed

Reply:
legacy guava support

Reply:
since we now shade newer gson no need of this